pkg/operator/ceph/operator.go

```diff
@@ -191,7 +194,7 @@ func (o *Operator) Run() error {
 	// Signal handler to stop the operator
 	for {
 		select {
-		case <-signalChan:
+		case <-stopContext.Done():
 			logger.Info("shutdown signal received, exiting...")
```

Comment:
The [example](https://pkg.go.dev/os/signal#example-NotifyContext) in the docs suggests calling `stopFunc()` function here to restore the default behavior of the signal as soon as possible. Do we need that?  Not sure why call to `defer stopFunc()` above won't be enough.

Reply:
Interesting I'll add that then.

Reply:
Snippet from a conversation on Gophers slack channel: 

```It’s usually common to defer call cancel() (or, in this case, stop()) to make sure you free resources when you’re done. Calling it multiple times has no side-effects, but forgetting to call mean you’ve a resource leak.
In the case of signal.NotifyContext, you should call stop() as soon as you want your process to start processing signals again, instead of ignoring them. (edited) 
3:05
(of course, if you’re terminating your program, it doesn’t matter, but you might want to use this for other things, like to receive a SIGHUP signal and reload a configuration, etc.) ```

Reply:
Ok so `defer` was enough it seems.

Reply:
yes. Since we are terminating, defer should be enough.